### DISCUSSION TITLE
Semantic-release not recognizing ssh keys on circleci

### DISCUSSION THREAD

**Discussion by machine-netizen** [semantic-release/semantic-release#1965] [Configuration] [answered]
For whatever reason, semantic-release isn't recognizing my ssh keys on circleci. 
I get the error `ENOGHTOKEN No GitHub token specified` even though I have deploy keys set up in circleci and I followed https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/git-auth-ssh-keys.md to add ssh keys during the ci job. 

Ci file:

`release:
    docker:
      - image: cimg/node:15.1
    steps:
      - checkout
      - add_ssh_keys:
          fingerprints:
            - 'user key'
            - 'deploy key'
      - run:
          name: deploy prod
          command: bash decrypt.sh && npx semantic-release`

Where decrypt.sh is the decryption steps in https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/git-auth-ssh-keys.md

I could be missing something really obvious but I don't think so

**Comment by travi** [comment, marked as answer]
the error is for the github token used for api access to github. this is different than the credentials used for git operations, like an ssh key. if you intend to use the github plugin, you need to supply a github token as well
